deepchem/models/torch_models/torch_model.py

```diff
@@ -347,7 +347,8 @@ def fit_generator(self,
                       max_checkpoints_to_keep: int = 5,
                       checkpoint_interval: int = 1000,
                       restore: bool = False,
-                      variables: Optional[List[torch.nn.Parameter]] = None,
+                      variables: Optional[Union[List[torch.nn.Parameter],
```

Comment:
Add on a discussion of this to the docs explaining the differences between `List[torch.nn.Parameter]` and `torch.nn.ParameterList`

Reply:
done